bundles/io/org.eclipse.smarthome.io.transport.mqtt/.gitignore

```diff
@@ -0,0 +1 @@
+OSGI-INF/*.xml
```

Comment:
PLease create the OSGI-INF folder and place the .gitignore inside with content *.xml.